cli/command/environment.go

```diff
@@ -192,10 +208,8 @@ func (e *EnvironmentCommand) update(c *cli.Context) error {
 		return err
 	}
 
-	req := models.UpdateEnvironmentRequest{
-		EnvironmentID:   id,
-		MinClusterCount: &minClusterCount,
-	}
+	req := models.UpdateEnvironmentRequest{EnvironmentID: id}
```

Comment:
This struct should not have the `EnvironentID` field

Reply:
? can you elaborate - not having environment id here would require changing the client interface for all the entities.